Automattic/jetpack
# Admin Page: Rename constants.js to constants/urls.js so it does not conflict with module resolution

Fixes #8104 

More info in [https://github.com/webpack/webpack/issues/4159](https://github.com/webpack/webpack/issues/4159).

#### Changes proposed in this Pull Request:

* Moves the file `_inc/client/constant.js` to `_inc/client/constants/urls.js`
* Updates references to that file from others in their import statements.

#### Testing instructions:

* checkout this branch
* Build it
* Visit the admin page
* Confirm that you can click the "Ask a question" button in any of the Happinness Support cards and be led to the support pages.
* Confirm that images show properly in plans tab


#### Before

![image](https://user-images.githubusercontent.com/746152/32611789-4c02d344-c545-11e7-8f41-abc6a68a4a0c.png)

#### After

![image](https://user-images.githubusercontent.com/746152/32611801-56145722-c545-11e7-8e53-fa8de8d24218.png)
